tasks/libs/ciproviders/github_api.py

```diff
@@ -519,14 +519,29 @@ def get_token_from_app(app_id_env='GITHUB_APP_ID', pkey_env='GITHUB_KEY_B64'):
         auth_token = integration.get_access_token(install_id)
         print(auth_token.token)
 
-    def create_label(self, name, color, description=""):
+    def create_label(self, name, color, description="", ignore_existing=False):
         """
         Creates a label in the given GitHub repository.
         """
-        return self._repository.create_label(name, color, description)
+        import github
```

Comment:
We already import `GithubException` above so this should not be necessary.

https://github.com/DataDog/datadog-agent/blob/d8fcc28fc89f0e8a3c61ffb3755d06f305a3ee84/tasks/libs/ciproviders/github_api.py#L21